cmd/tnf/check/check.go

```diff
@@ -0,0 +1,19 @@
+package check
+
+import (
+	"github.com/spf13/cobra"
+	imagecert "github.com/test-network-function/cnf-certification-test/cmd/tnf/check/image_cert_status"
+)
+
+var (
+	checkCmd = &cobra.Command{
+		Use:   "check",
```

Comment:
I feel check is to vague, what about certification?

Reply:
I believe that `preflight` also uses the "check":

![image](https://github.com/test-network-function/cnf-certification-test/assets/4563082/974fb695-b6dd-4816-b246-cf70300f3335)

Reply:
https://github.com/redhat-openshift-ecosystem/openshift-preflight#usage